libr/anal/fcn.c

```diff
@@ -1797,9 +1799,11 @@ R_API RAnalFunction *r_anal_fcn_find_name(RAnal *anal, const char *name) {
 /* rename RAnalFunctionBB.add() */
 R_API int r_anal_fcn_add_bb(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut64 size, ut64 jump, ut64 fail, int type, RAnalDiff *diff) {
 	RAnalBlock *bb = NULL, *bbi;
+	ut8 *bbuf;
 	RListIter *iter;
 	bool mid = false;
 	st64 n;
+	int old_num_bbs;
```

Comment:
And here

Reply:
no longer relevant due to dcd1f66ae9ee3cdadee69419ac4a2b29098e6c0b